de.metas.contracts/src/test/java/de/metas/contracts/impl/ContractChangeBLTest.java

```diff
@@ -164,5 +191,20 @@ private void assertSubscriptionProgress(@NonNull final I_C_Flatrate_Term flatrat
 				.filter(progress -> progress.getEventDate().after(flatrateTerm.getMasterEndDate()))
 				.peek(progress -> assertThat(progress.getContractStatus()).isEqualTo(X_C_SubscriptionProgress.CONTRACTSTATUS_Running));
 	}
+	
+	private void assertClosedFlatrateTerm(@NonNull final I_C_Flatrate_Term flatrateTerm)
+	{
+		assertThat(flatrateTerm.getDocStatus()).isEqualTo(X_C_Flatrate_Term.DOCSTATUS_Completed);
```

Comment:
shouldn't the term have docstaus *closed* afterwards?

Reply:
No, I still completed. Just the ConstractStatus is Quit. In the end, we using the same process as for general cancelling.